test/Sync.t.sol

```diff
@@ -125,7 +130,7 @@ contract SyncTest is Test, Deployers, GasSnapshot {
 
         // Sync has not been called.
         vm.expectRevert(Reserves.ReservesMustBeSynced.selector);
-        manager.getReserves(currency0);
+        poolStateViewer.getReserves(currency0);
 
         manager.sync(currency0);
         assertEq(manager.getReserves(currency0), currency0Balance);
```

Comment:
why is `poolStateViewer.getReserves` needed for reverts but `manager.getReserves` needed for seeing the return value?

Reply:
> We cannot use the library directly, vm.expectRevert uses the next call, which in that case would be manager.exttload within the library. So the expected revert is not thrown in the next call but within the library, that's why I introduced the viewer contract